ansible/group_vars/all.yaml

```diff
@@ -0,0 +1,26 @@
+ethereum_helm_chart_url: https://skylenet.github.io/ethereum-helm-charts
+
+# for k3s traefik is installed by default
+ingress_controller: traefik
+
+# Change accordingly -> xdai or sokol
+ethereum_network: xdai
+
+# k8s configuration
+k8s_kubeconfig: /etc/rancher/k3s/k3s.yaml
+
+# envinroment deployment prd (xdai) or stg (sokol)
+env: prd
+
+# allowed users to connect to the xdainode base64 encoded
+allowed_users:
```

Comment:
Who are these users?

Reply:
these are whoever want to have access to the node